test/keyboard-navigation.html

```diff
@@ -812,7 +783,7 @@
 
             expect(getFocusedCellIndex()).to.equal(1);
             const columnFocusedCell = grid.shadowRoot.activeElement._column;
-            expect(columnFocusedCell.hidden).to.be.false;
+            expect(columnFocusedCell.hidden).not.to.be.true;
```

Comment:
`false` is not the default value for `hidden`, which seems to be undefined. To be safe I switched the assertion the check against `true`.